languages/javascript/reference/info/scope.md

```diff
@@ -0,0 +1,7 @@
+## Scope / Visibility
+
+Most languages have some concept of encapsulation through scope and visibility.
+Tracks should explain the language-concepts and keywords that their track uses (e.g. `public`, `private`, `export`) and what the normal way of using those scopes are in their language. An exercise may be appropriate, but providing examples in stubs early on may avoid overwhelming the student.
+
+[Global Scope](#global-scope)
```

Comment:
I would also make this a sub headline (it will get this hash-automagically on GitHub).